include/dkm_parallel.hpp

```diff
@@ -87,6 +94,7 @@ std::vector<std::array<T, N>> random_plusplus_parallel(
 		std::discrete_distribution<input_size_t> generator(distances.size(), 0.0, 0.0, [&distances, &i](double) { return distances[i++]; });
 #endif
 		means.push_back(data[generator(rand_engine)]);
```

Comment:
It seems I cannot add a suggestion on multiple unmodified lines, but anyway. I think lines 88-96 (except for the preprocessor directives), should be indented since they now fall into the `else` branch.